packages/ffe-form-react/src/PhoneNumber.js

```diff
@@ -18,14 +20,42 @@ export default class PhoneNumber extends React.Component {
             onCountryCodeBlur,
             onNumberBlur,
             disabled,
-            countryCodeInvalid,
-            numberInvalid,
+            countryCodeFieldMessage,
+            numberFieldMessage,
+            countryCodeAndNumberFieldMessage,
             className,
+            extraMargin = true,
```

Comment:
Den burde være i `defaultProps` litt lengre ned i filen. Kan bli forvirring å dele opp defaults på to sted.